etc/checkstyle-custom_checks.xml

```diff
@@ -162,5 +163,5 @@
         <!--The system property is set via Maven-->
         <property name="file" value="${checkstyle.suppressions.file}"/>
     </module>
-
+    <module name="SuppressWarningsFilter" />
```

Comment:
this used to be in the checkstyle config but was lost at some point, without these changes checkstyle is not honouring `@SuppressWarnings` annotations